charts/fybrik/values.yaml

```diff
@@ -204,7 +204,8 @@ manager:
     # The name of the service account to use
     name: manager
 
-  podAnnotations: {}
+  podAnnotations:
+    sidecar.istio.io/inject: "true"
```

Comment:
now by default you always add this annotation, I'm not sure that it is right

Reply:
I put the label in a comment.